docs/source/examples/tabular/checks/methodology/source/plot_train_test_samples_mix.py

```diff
@@ -2,41 +2,42 @@
 """
 Train Test Samples Mix
 **********************
+This notebook provides an overview for using and understanding the Train Test Samples Mix check.
+
+**Structure:**
+
+* `Why samples mix are unwanted? <#why-samples-mix-are-unwanted>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+Why samples mix are unwanted?
+=============================
+Samples mix is a situation when train and test have joint samples. While using the test dataset for evaluating the
+model, if we have samples from the train dataset which was used for training the model, then the resulting metrics will
+be biased and won't reflect the performance we will get in a real life scenario. Therefore, it's important to always
+avoid samples mix.
```

Comment:
The second sentence ("while...") is long and complicated and I'm not sure what you mean by it. Can you try and rephrase it (and maybe break it into 2?)